pkg/statuschecker/types/built_in.go

```diff
@@ -66,23 +67,65 @@ func alwaysReady(_ runtime.Object) (statuschecker.ObjectStatusResult, error) {
 	return statuschecker.ObjectStatusReady{}, nil
 }
 
+func getDeploymentCondition(status apps.DeploymentStatus, condType apps.DeploymentConditionType) *apps.DeploymentCondition {
```

Comment:
Would be better to either take the deployment or a slice of conditions as the first argument - either simplifies or makes it more generic. Currently it is somewhere in the middle without any reason/benefit.

Reply:
Done